terraform/development/shared/main.tf

```diff
@@ -153,6 +153,16 @@ module "database" {
   vpc_id                      = module.networking.vpc_id
 }
 
+module "deployment_role" {
+  source = "../../modules/terraform_deployment"
+
+  prefix = local.prefix
+  assume_from_role_arns = [
+    "arn:aws:iam::815624722760:role/developer",
+    "arn:aws:iam::815624722760:role/core-infrastructure-repo"

```

Comment:
We've got a future ticket already to get some outputs from meta - leaving making this not hardcoded to be part of that.